src/features/events/components/EventPopper/ParticipantAvatars.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Avatar, AvatarGroup, Box } from '@mui/material';
+
+import theme from 'theme';
+import ZUIPersonHoverCard from 'zui/ZUIPersonHoverCard';
+import {
+  ZetkinEventParticipant,
+  ZetkinEventResponse,
+} from 'utils/types/zetkin';
+
+interface ParticipantAvatarsProps {
+  orgId: number;
+  participants: ZetkinEventParticipant[] | ZetkinEventResponse[];
+}
+
+const ParticipantAvatars = ({
+  participants: list,
+  orgId,
+}: ParticipantAvatarsProps) => {
+  return (
+    <Box alignItems="center" display="flex" flexWrap="wrap" gap={0.5}>
+      <AvatarGroup
+        max={18}
+        sx={{
+          /* surplus chip styling */
```

Comment:
There's something about the look of this chip to me that feels a bit off. I think it's the lack of padding on the left and right?

![image](https://github.com/zetkin/app.zetkin.org/assets/550212/6e2d5756-fad9-4912-b521-65e5131d2320)